hs_core/templates/pages/genericresource.html

```diff
@@ -57,7 +59,13 @@ <h2 class="page-title">{{ title }}</h2>
                             {% else %}<span>{{ cr.name }}</span><span class="comma">,&nbsp</span>{% endif %}
```

Comment:
Sorry for my ignorance on HTML, I did not see where this entity is missing a ";" This code is added probably by @pkdash or @mauriel133 or someone else. @pkdash @mauriel133 Do you see Calvin's point here and can make a fix quickly or let me know where I should add ";" to make the fix?